bundle/downstream/manifests/quayregistries.crd.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   group: quay.redhat.com
   names:
-    kind: QuayRegistry
+    kind: QuayRegistryf
```

Comment:
Isn't this `f` at the end of `QuayRegistryf` an error? Should be just `QuayRegistry` as `kind`, no?

Reply:
yes, seems like a typo, no?